ibu-imager/hack/clean.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # 1) cleanup backup dir
-rm -rf /var/tmp/backup /var/tmp/checks
+rm -rf /var/tmp/backup /var/tmp/checks /var/tmp/backupCertsDir
```

Comment:
We should not need this script after having cleanup operation or what i miss?

Reply:
Agree, will remove it as part of this PR